Frameworks/AEPSwiftUI/Sources/AEPSwiftUI+PublicAPI.swift

```diff
@@ -14,8 +14,6 @@ import AEPMessaging
 import AEPServices
 import Foundation
 
-@objc(AEPSwiftUI)
-@objcMembers
```

Comment:
thank you for removing this.
Can we remove one last one in the class `ContentCardUIError` ?